modules/ROOT/pages/dataweave-selectors.adoc

```diff
@@ -48,6 +41,618 @@ xref:dataweave-cookbook-extract-data#selector_metadata[Extract Data].
+---
+payload.name
+----
+
+==== Output
+
+The script outputs an array that contains the values of the input objects
+with the key "name": "Arg" and "Japan".
+
+[source,json, linenums]
+----
+["Arg", "Japan"]
+----
+
+[[namespace_example1]]
+== Namespace Selector
+
+The DataWeave namespace selector supports the use of a name that matches a
+namespace or a local part. To match a namespace, the name must match the
+namespace and the local part. If the name only matches the local part, the
+selector will match all namespaces with that local part, regardless of the
+namespace.
+
+Note that XML namespaces and their prefixes are defined in the `xmlns` attribute
+of an XML element. For example, the element
+`<h:table xmlns:h="http://www.w3.org/TR/html4/"/>` defines a namespace that
+assigns the prefix `h` to the namespace  `http://www.w3.org/TR/html4/`.
+All elements in that namespace contain the `h` prefix. For example, in the
+element `<h:table/>`, `h` is the prefix for the namespace, and `table` is the
+local part.
+
+=== Example
+
+This example selects values from XML elements using the namespace and local part.
+
+==== Input
+
+The input payload contains XML elements, `h:table` and `f:table`, that have
+different namespaces but the same local names (`table`).
+
+[source,xml, linenums]
+----
+<root>
+
+  <h:table xmlns:h="http://www.w3.org/TR/html4/">
+    <h:tr>
+      <h:td>Apples</h:td>
+      <h:td>Bananas</h:td>
+    </h:tr>
+  </h:table>
+
+  <f:table xmlns:f="https://www.w3schools.com/furniture">
+    <f:tr>
+      <f:name>African Coffee Table</f:name>
+      <f:width>80</f:width>
+      <f:length>120</f:length>
+    </f:tr>
+  </f:table>
+
+</root>
+----
+
+[[namespace_variable]]
+==== Source
+
+The DataWeave script selects XML content from specified namespaces. The
+script's header creates namespace (`ns`) variables `html` and `furniture`
+to store namespaces from the input payload. To select the children of `h:table`
+into element `a` and the children of `f:table` elements into element `b`, the
+script uses the `html` and `furniture` namespace variables with a
+namespace selector (`#`) and local part (`table`).
+
+[source,DataWeave, linenums]
+----
+%dw 2.0
+output application/xml
+ns html http://www.w3.org/TR/html4/
+ns furniture https://www.w3schools.com/furniture
+---
+root: {
+     a: payload.root.html#table,
+     b: payload.root.furniture#table
+ }
+----
+
+==== Output
+
+The script outputs children of the `h:table` element under element `a` and
+the children of the `f:table` under element `b`.
+
+[source,xml, linenums]
+----
+<?xml version='1.0' encoding='UTF-8'?>
+<root>
+  <a>
+    <h:tr xmlns:h="http://www.w3.org/TR/html4/">
+      <h:td>Apples</h:td>
+      <h:td>Bananas</h:td>
+    </h:tr>
+  </a>
+  <b>
+    <f:tr xmlns:f="https://www.w3schools.com/furniture">
+      <f:name>African Coffee Table</f:name>
+      <f:width>80</f:width>
+      <f:length>120</f:length>
+    </f:tr>
+  </b>
+</root>
+----
+
+=== Example
```

Comment:
I wonder if we can reword these repeated ===Example titles so that they are specific to their use cases....